prom-keycloak-proxy/overlays/nerc-ocp-obs/services/prom-keycloak-proxy/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: prom-keycloak-proxy
+  labels:
+    app: prom-keycloak-proxy
+    deployment: prom-keycloak-proxy
```

Comment:
Using `commonLabels` in your `kustomization.yaml` will also keep these labels and `spec.selector` in sync with the labels in the deployment.

Reply:
I applied this suggestion, thanks.